<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaHome</class>
 <widget class="QWidget" name="VistaHome">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>389</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>576</width>
    <height>389</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestore Centro Sportivo PRO</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">image: url(:/immagini/sfondo.jpg);</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QWidget" name="widget_4" native="true">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(69, 162, 0);button</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QWidget" name="widget_2" native="true">
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="4" column="1">
                <widget class="QPushButton" name="assegnamento_button_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(69, 162, 0);
background-color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="text">
                  <string>Assegnamento</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QPushButton" name="iscrizioni_button_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(69, 162, 0);
background-color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="text">
                  <string>Iscrizioni</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QPushButton" name="clienti_button_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(69, 162, 0);
background-color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="text">
                  <string>Clienti</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QPushButton" name="prenotazioni_button_2">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>18</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color: rgb(69, 162, 0);
background-color: rgb(0, 0, 0);</string>
                 </property>
                 <property name="text">
                  <string>Prenotazioni</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
